includes/blocks/class-sensei-block-quiz.php

```diff
@@ -24,7 +24,7 @@ public function __construct() {
 			[
 				'render_callback' => [ $this, 'render_quiz' ],
 			],
-			Sensei()->assets->src_path( 'blocks/quiz/quiz-block' )
```

Comment:
@merkushin Would you mind clarifying what this line fixes?

Reply:
It registers the exact block.
https://github.com/Automattic/sensei/pull/6994#issuecomment-1936618063

Reply:
Sorry for the intromission, but I think this specific change reason is that `blocks/quiz/block.json` doesn't exist and the correct path would be `blocks/quiz/quiz-block`. It points to the folder where the `block.json` lives.

Reply:
> It points to the folder where the block.json lives

That's right. Just want to add here that we have several nested blocks for the quiz block. If we would like to see translations for them in the editor (in the inspector, for example), we need to register each of them separately.

Reply:
I put this line back in and re-tested quiz translations and they are working. The question description is also being displayed now. 🥳